src/fragments/lib/analytics/android/record.mdx

```diff
@@ -101,6 +101,86 @@ RxAmplify.Analytics.flushEvents();
 </Block>
 </BlockSwitcher>
 
+## Authentication events
+Indicate how frequently users authenticate with your application.
+
+On the **Analytics** page, the **Users** tab displays charts for **Sign-ins, Sign-ups, and Authentication failures**.
+
+To learn how frequently users authenticate with your app, update your application code so that Pinpoint receives the following standard event types for authentication:
+
+ - `_userauth.sign_in`
+ - `_userauth.sign_up`
+ - `_userauth.auth_fail`
+
+You can report authentication events by doing either of the following:
+
+ - Managing user sign-up and sign-in with Amazon Cognito user pools.
+
+  Cognito user pools are user directories that make it easier to add sign-up and sign-in to your app. As users authenticate with your app, Cognito reports authentication events to Pinpoint. For more information, see [Using Amazon Pinpoint Analytics with Amazon Cognito User Pools](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-pinpoint-integration.html) in the _Amazon Cognito Developer Guide_. Also update **awsconfiguration.json** by adding the `PinpointAppId` key under `CognitoUserPool`.
+
+  ```json
+  "CognitoUserPool": {
+      "Default": {
+          "PoolId": "<poolid>",
+          "AppClientId": "<appclientid>",
+          "Region": "<region>",
+          "PinpointAppId": "<pinpointappid>"
+     }
+  }
+  ```
+
+- Reporting authentication events by using the Pinpoint client that's provided by the AWS Mobile SDK for iOS or Android.
```

Comment:
```suggestion
- Manually recording authentication events using the `recordEvent()` API.
```

Reply:
Can we create fragements in code for these text and reuse them in particular sections. So that we can use the same text for v1/v2/ios/android